src/core/symbology/qgsfillsymbollayer.h

```diff
@@ -1734,13 +1734,19 @@ class CORE_EXPORT QgsRandomMarkerFillSymbolLayer : public QgsFillSymbolLayer
 {
   public:
 
+    enum CountMethod
+    {
+      AbsoluteCount, //!< The point count is used as an absolute count of markers
+      DensityBasedCount, //!< The point count is part of a marker density count
+    };
```

Comment:
Just a suggestion: as count might be misleading, what about Threshold for the enum name and something like MaximumDensity for the second enum variable?

Reply:
I don't really see threshold as a better wording here. But that might just be the fact that I'm not a native English speaker. Let's see what Nyall comes up with.

Reply:
I don't think threshold works either -- it suggests it's going to render some number up to the count, yet it will **always** render exactly that number of points